NOAA-FIMS/FIMS
# Feat: add test case documentation

*What is the feature?*
This PR addresses issue from NOAA-FIMS/collaborative_workflow#65. It adds milestone 1 test cases documentation under the tests/ folder.

*How have you implemented the solution?*
- added milestone1_test_cases.md template to the repo
- added milestone1_test_cases.md to .gitbuildignore file 
- also renamed the test-place-holder.R without changing anything inside the file

*Does it impact any other area of the project?*
No.

*Does this change impact the model input or output?*
No.

*How to test this change*
All the GH Actions workflow passed except the R cmd check. The failed run was not caused by the changes in this PR.
